src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -5169,7 +5169,7 @@ instruct storeImmP0(memory mem, immP0 zero)
   ins_pipe(ialu_mem_reg);
 %}
 
-// Store NULL Pointer, mark word, or other simple pointer constant.
+// Store nullptr Pointer, mark word, or other simple pointer constant.
```

Comment:
Keep `NULL`

Reply:
I made this one Null too since we don't want to find NULL in the code because it'll look like this change backtracked.